workers/worker_scheduler.rb

```diff
@@ -0,0 +1,32 @@
+require "rufus-scheduler"
+
+module FastlaneCI
+  # Class that handles the scheduling for fastlane.ci workers
+  class WorkerScheduler
+    # Sleep in seconds
+    attr_accessor :sleep_interval
+    # Ex. '5 0 * * *' do something every day, five minutes after midnight
+    # (see "man 5 crontab" in your terminal)
+    attr_accessor :cron_time
```

Comment:
I think `cron_schedule` might be a little more descriptive as a name

Reply:
done.